events/0316_git.html

```diff
@@ -0,0 +1,215 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+
+    <title>Intro to Version Control: Git, Github &amp; the Terminal | Girl Develop It Ann Arbor</title>
+    <meta name="author" content="Girl Develop It Ann Arbor">
+
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+    <link rel="stylesheet" href="css/reveal.min.css">
+    <link rel="stylesheet" href="css/theme/gdiaa.css" id="theme">
+    <link rel="stylesheet" href="lib/css/rainbow.css">
+    <link rel="stylesheet" href="css/print/pdf.css" media="print">
+
+    <script src="lib/js/head.min.js"></script>
+
+    <!--[if lt IE 9]>
+    <script src="lib/js/html5shiv.js"></script>
+    <![endif]-->
+</head>
+<body>
+    <div class="reveal">
+        <!-- Any section element inside of this container is displayed as a slide -->
+        <div class="slides">
+
+            <!-- Title -->
+            <section>
+                <img src="images/gdi_logo_badge.png" class="img--bare" height="450px" />
+                <div class="box--small">
+                    <h3 class="event__title">Intro to Version Control<br/><small><span class="green">Git, GitHub, &amp; the Terminal</span></small></h3>
+                    <p><small>James York &nbsp; | &nbsp; @kroysemaj &nbsp; | &nbsp; @gdiannarbor &nbsp; | &nbsp; #GDIA2 &nbsp; | &nbsp; #git-intro</small></p>
```

Comment:
I've started removing event specific hashtags b/c I think they deter from the main #GDIA2 tag, so let's get rid of #git-intro wherever it exists